LICENSE

```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright 2021 Twilio
```

Comment:
Ideally copyright and version changes would be in a separate commit from the fix.